xilem_core/src/any_view.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2024 the Xilem Authors
+// Copyright 2023 the Xilem Authors
```

Comment:
Does it make sense to add ranges (i.e. 2023-2024) here? I think that's more accurate?

Reply:
I don't think we want to do that

Afaik, the years don't really mean much, and are much more of a formality.